src/core_modules/capture-core/components/WidgetNote/NoteSection/NoteSection.tsx

```diff
@@ -42,7 +33,7 @@ const styles = {
     },
     notesWrapper: {
         maxHeight: 400,
-        overflowY: 'auto',
+        overflowY: 'auto' as const,
```

Comment:
It looks like you added `as const` to fix the error `Type 'string' is not assignable to type 'OverflowYProperty | undefined'`. Instead, I want to fix this error by using `Readonly` for styles. The fix should look like this: `const style: Readonly<any>`